src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function fetchMemeFile() {
+  const responseFromServer = await fetch('/memes');
+  const textFromResponse = await responseFromServer.text();
+  const memeContainer = document.getElementById('memes-container');
+  memeContainer.innerHTML = textFromResponse;
+}
```

Comment:
For the future, this might make more sense in whatever js file that's specific to the "see all memes" page (instead of having a shared js file for the homepage and see all memes page) but for now we can leave it here.